draft-parecki-oauth-client-id-metadata-document.md

```diff
@@ -131,8 +131,7 @@ This specification defines the client identifier as a URL with the following
 restrictions. Client identifier URLs MUST have an "https" scheme, MUST contain a
 path component, MUST NOT contain single-dot or double-dot path segments, MUST
 NOT contain a fragment component and MUST NOT contain a username or password
-component. Client identifier URLs MAY contain a query string component and MAY
-contain a port.
+component. Client identifier URLs MAY contain a port.
```

Comment:
Was it intentional to just remove the mention of a query string component, rather than say "MUST NOT" include a query string component?

Reply:
hmm, I'm not sure what would be best? maybe "SHOULD NOT" ? The idea is to discourage usage as much as possible.

Reply:
"SHOULD NOT" would be in line with the comment here https://github.com/aaronpk/draft-parecki-oauth-client-id-metadata-document/issues/23#issuecomment-2575500904

Reply:
Let's add "SHOULD NOT" here and merge this in for publishing version -02 today